### DISCUSSION TITLE
FYI: GFX focus function requires shutter button focus

### DISCUSSION THREAD

**Discussion by edcooley** [gkoh/furble#99] [General]
FYI: The focus function was not working for me as I don't use the shutter button for focus relying on the AF button.  Setting focus on the shutter release fixed it.

**Comment by gkoh** [comment]
@edcooley Oh, this is an interesting discovery. 
I never considered it might be dependent on the camera's focus configuration.
I'll have to play around with this, thanks!

**Reply by matthudsonau** [reply]
I'll have to check on mine, but I'm pretty sure it's dependant on the focus lever setting as well
